versioned_docs/version-0.6.3/fishjam_cloud.md

```diff
@@ -3,11 +3,10 @@ id: fishjam_cloud
 slug: /
 ---
 
-# Fishjam goes to the Cloud!
+# Fishjam goes to the Cloud
```

Comment:
I think it will be enough to just update current version of docs. If we would like to make sure all old docs use proper naming - it would take too much time.

Reply:
Gonna merge it this way as it is already done. But 0.6.3 is actually the default site that is rendered when a user views this website (aka current version